data/hai/unfettered jobs.txt

```diff
@@ -322,3 +401,408 @@ mission "Unfettered Big Medicals Rush"
+					goto where
+				`	"Can't you just make your own batteries?"`
+					goto why
+			`	"Well, as you have probably noticed, a lot of Hai use Ion Cannons in order to disable ships without doing too much damage to the crew. That in itself is a very good thing, but it also means that we need to actively work against it. Bigger batteries mean the guns will be less scrambled, and dimnishes the risk of being completely stranded in space."`
+				goto choose
+			label where
+			`	"These batteries are pretty big, so I would expect them to mostly be in big outfitters, such as Markerplace. They should not be too hard to find."`
+				goto choose
+			label why
+			`	"That is what we usually do, except that the need for those has increased recently, and we lack sufficient production to keep up with it. So we ask Hai Sympathizers and privateers like yourself to get them for us when we are in big need. Our navy must not seem weak or the Hai could use that opportunity to strike."`
+				goto choose
+	on complete
+		outfit "Hai Valley Batteries" -1
+		payment 500000
+		dialog `You bring the batteries to the deck workers, who seem to be in a hurry to put it to use, "About time! Would it have been any later it would have probably been too late! Here are your <payment>."`
+
+
+mission "Unfettered Bring Ion Cannons"
+	job
+	name "Bring Ion Cannons"
+	description "Bring 2 Ion Cannons to the Unfettered on <destination>. They are willing to offer you <payment> for it."
+	source
+		attributes "unfettered"
+	destination "Darkcloak"
+	to offer
+		random < 5
+		has "Unfettered Jump Drive 1: done"
+	on offer
+		conversation
+			`Seeing this peculiar request about Ion Cannons, which the Unfettered do have a bunch of, you go and look for a captain who made this job request, to ask them why they use those weapons. It does not take long to find them, and it seems like they are willing to answer your questions, at least partly, "I understand why you're wondering this, <first>... May I call you <first>? There are not many non pirates that come and supply us with jump drives, if at all, and that is how I know your name."`
+			choice
+				`	"Sure."`
+					goto sure
+				`	"I prefer we stick to a professional relation, call me <last>."`
+			`	"Very well, not that it matters much anyway."`
+			label sure
+			`	"You see, if we use Ion it is not really for the same reason as the other Hai do, to hurt others of our kind as little as possible, although that is a noble cause indeed, it is not being noble that brings you food," she laughes a bit, and then comes back to a serious attitude, "No. If we do it, it is because it forces the Hai to use more batteries on their ships, as to not be disabled too easily, or defenceless because of weapon scrambling. So, we do not use ion weapons most of the time, but sometimes we surprise them with a big amount of them. Expecting this, the Hai are always forced to keep a good amount of batteries, and, thus, to have less guns, shields..."`
```

Comment:
This paragraph could use some more work; the first part should definitely be rephrased.